### DISCUSSION TITLE
Is there an easy way to put in a placeholder in the search textbox option?

### DISCUSSION THREAD

**Discussion by frscott** [quarto-dev/quarto-cli#7599] [Q&A] [answered]
### Description

````qmd
website:
  search:
    location: navbar
    type: textbox
````

I would like to put in a placeholder. Is the only way to do it if I insert some JavaScript in the header?

**Comment by mcanouil** [comment]
See:
- https://github.com/quarto-dev/quarto-cli/discussions/6564

**Reply by frscott** [reply, marked as answer]
Thanks. I couldn't find that question when I was searching. Do you think adding an option to put in a placeholder could be considered for a search textbox enhancement? Here is my solution that also works for future folks looking into this.

````qmd
website:
  search:
    location: navbar
    type: textbox

format:
  html:
    include-in-header:
      - text: |
          <script>window.onload = function() {document.getElementById("autocomplete-0-input").placeholder = "Search Drugs";};</script>

````

![searchbox](https://github.com/quarto-dev/quarto-cli/assets/59030252/73fdfb6c-2fa0-4922-bb9c-936a8958d994)

**Reply by mcanouil** [reply]
cc @dragonstyle

**Reply by dragonstyle** [reply]
Thanks for the suggestion, this will be supported In our next pre-release build. You can control using:

```
language: 
  search-text-placeholder: "Search Contents"
```
